src/kenigsberg/MorseCode/MorseCodeTest.java

```diff
@@ -0,0 +1,34 @@
+package kenigsberg.MorseCode;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+
+class MorseCodeTest
+{
+    @Test
+    public void testToMorseCode()
```

Comment:
`toMorseCode()`
You don't need to put the word test in front of the method name.